resolwe/flow/managers/base.py

```diff
@@ -134,38 +153,67 @@ def load_executor(self, executor_name):
                 # If there's some other error, this must be an error in Django
                 raise
 
-    def load_exprengines(self, exprengine_list):
-        """Load expression engines."""
-        exprengines = {}
+    def _load_engines(self, class_name, base_module, engines):
```

Comment:
Is there an easy way to move this function out of the class and also use it to load manager and executor?